src/custom/ecospheres/views/HomeView.vue

```diff
@@ -56,6 +59,54 @@ useHead({
         </div>
       </div>
     </section>
+    <section class="fr-container">
+      <ul class="fr-grid-row fr-grid-row--gutters fr-py-12v" role="list">
+        <li class="fr-col-12 fr-col-md-6">
+          <DsfrCard
+            :no-arrow="true"
+            :title="'Explorer'"
+            :description="'Accédez à un ensemble de données environnementales'"
```

Comment:
```suggestion
            title="Explorer"
            description="Accédez à un ensemble de données environnementales"
```
Ce n'est pas nécessaire d'avoir des propriétés dynamiques ici vu qu'on a seulement des strings.